lib/client.js

```diff
@@ -146,6 +146,9 @@ exports.createClient = function(reference, name) {
       exchangePrefix:   reference.exchangePrefix || ''
     }, _defaultOptions);
 
+    // bug 1310643 - remove possible trailing slash from baseUrl
+	this._options.baseUrl = this._options.baseUrl.replace(/\/$/, '')
```

Comment:
Looks great, but I see you've been coding a lot of golang, because this uses tabs for indentation :)